src/query/__tests__/RelayFragmentPointer-test.js

```diff
@@ -63,6 +64,40 @@ describe('RelayFragmentPointer', () => {
       );
     });
 
+    it('creates a wrapped fragment pointer with number arg', () => {
```

Comment:
I suspect that the tests you want to write will live in `RelayQueryRoot-test.js`. Start from `it('returns the identifying argument type', …)` and work outward to assert that non-string identifying arguments are permitted on root fields.